pkg/apis/migration/migration.go

```diff
@@ -70,6 +71,10 @@ type VolumeMigrationService interface {
 	// Returns an error if not able to retrieve VolumePath.
```

Comment:
nit: extend the function doc to mention about CR creation and Volume Registration.

That way we can clearly understand the difference between  `GetVolumePath` & `GetVolumePathFromMigrationServiceCache`

Reply:
Updated the comment.